extensions/eol-runtime-scanner/www/styles.css

```diff
@@ -13,6 +13,77 @@
   margin-top: -10px;
 }
 
+/* Version selector styles */
+.version-selector {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 4px;
+  padding-top: 5px;
+}
+
+/* ActionButton version */
+.btn-version {
+  padding: 2px 6px;
+  font-size: 0.8rem;
+  font-family: "Fira Mono", Consolas, Monaco, monospace;
+  border: 1px solid #ddd;
+  background: #f8f9fa;
+  border-radius: 3px;
+  margin-right: 0;
+  margin-bottom: 4px;
+  transition: all 0.15s ease-in-out;
+}
+
+/* Server version styling */
+.btn-version.server-version {
+  /* Regular font weight for server versions */
+}
+
+/* Non-server version styling - make it italic */
+.btn-version:not(.server-version) {
+  font-style: italic;
+}
+
+/* Selected versions */
+.btn-version.selected {
+  background-color: #447099; /* Posit Blue */
+  color: white;
+  border-color: #447099;
+}
+
+/* The specifically selected version */
+.btn-version.selected.max {
+  text-decoration: underline; /* Underline the max version */
+  text-underline-offset: 2px; /* Add some space between text and underline */
+}
+
+/* Add cursor pointer to btn-version for better UX */
+.btn-version {
+  cursor: pointer;
+}
+
+/* Hover effect for unselected buttons */
+.btn-version:hover:not(.selected) {
+  background-color: #e9ecef;
+  border-color: #ced4da;
+}
+
```

Comment:
Bootstrap sets `color: var(--bs-btn-hover-color)` here which is `#fff`. I'd recommend overriding that style so the hover state is readable.

![CleanShot 2025-06-23 at 13 55 49](https://github.com/user-attachments/assets/bf05e953-b46a-46ab-bb2e-651bb39b1479)

The below will keep the `color` you set for in a parent element
```suggestion
.btn-version:hover:not(.selected) {
  background-color: #e9ecef;
  color: inherit;
  border-color: #ced4da;
}

```

![CleanShot 2025-06-23 at 13 57 19](https://github.com/user-attachments/assets/54fe0b45-9839-4737-8636-786e0b04add8)